tests/Fixtures/Entity/Foo.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Fixtures\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="foos")
+ */
+class Foo
```

Comment:
Let's rename to `EntityWithRelations` (not a great name but not as generic as Foo).